package.json

```diff
@@ -30,13 +30,13 @@
     "appium-logger": "^2.0.0",
     "appium-support": "^2.0.0",
     "asyncbox": "^2.0.2",
-    "babel-runtime": "=5.8.24",
+    "babel-runtime": "=5.8.25",
```

Comment:
Let's keep this at 5.8.24 until we can go through all the modules. Sometimes mixing babel versions causes problems.